context-extraction/src/main/java/de/atb/context/persistence/context/ContextRepository.java

```diff
@@ -71,11 +70,12 @@ private ContextRepository() {
     }
 
     private synchronized void persistNamedContext(ContextContainer context) {
-        Store store = getStore(context.getBusinessCase());
-        Model named = SDBFactory.connectNamedModel(store, context.getIdentifier());
+// FIXME to fix
+/*        Store store = getStore(context.getBusinessCase());
+        Model named = TDBFactory.connectNamedModel(store, context.getIdentifier());
         named.removeAll();
         named.add(context);
-        named.close();
+        named.close();*/
     }
 
     private synchronized void persistReasonableContext(ContextContainer context) {
```

Comment:
Maybe check the return value of `new File(fileLocation).mkdirs();` in `persistRawContext` (line 90)?

Reply:
[x] done